docs/store/testing.md

```diff
@@ -108,3 +108,40 @@ describe('My Component', () => {
   });
 });
 ```
+### Testing selectors
+If you have selectors composed of many other selectors it may be useful to test them independent from the selectors they are composed of. Therefor selectors contain a reference to the projector function as `.projector`.
```

Comment:
Will you change this to "You can use the projector function used by the selector by accessing the `.projector` property`